docs/reference/juju-cli/list-of-juju-cli-commands/help-action-commands.md

```diff
@@ -0,0 +1,26 @@
+(command-juju-help-action-commands)=
+# `juju help-action-commands`
+> See also: [help](#help), [help-hook-commands](#help-hook-commands)
+
+## Summary
+Show help on a Juju charm action command.
+
+## Usage
+```juju help-action-commands [options] [action]```
+
+## Examples
+
+For help on a specific action command, supply the name of that action command, for example:
+
+        juju help-action-commands action-fail
+
+
+## Details
+
+Juju charms have access to a set of built-in helpers known as 'action-commands,'
```

Comment:
We should mention that actions can also run hook commands.

Reply:
Sorry Ian what do you mean by this?

Reply:
Resolved offline